frontend/ui/widget/dictquicklookup.lua

```diff
@@ -61,6 +61,25 @@ local highlight_strings = {
     unhighlight = _("Unhighlight"),
 }
 
+function DictQuickLookup:canSearch()
+    if self:isDocless() then
+        return false
+    end
+
+    if self.is_wiki then
+        if #self.wiki_languages > 1 then
```

Comment:
A small comment mentionning that the Search button is used to show Wikipedia languages would be nice (because I wondered what Wikipedia has to do with that :)